roofit/hs3/test/testRooFitHS3.cxx

```diff
@@ -44,9 +44,57 @@ TEST(RooFitHS3, RooArgusBG)
    RooAddPdf model("model", "g+a", RooArgList(signalModel, background), RooArgList(nsig, nbkg));
 
    std::string rootetcPath = gSystem->Getenv("ROOTSYS");
-   RooJSONFactoryWSTool::loadExportKeys(rootetcPath + "/etc/root/RooFitHS3_wsexportkeys.json");
+   RooJSONFactoryWSTool::loadExportKeys(rootetcPath + "/etc/RooFitHS3_wsexportkeys.json");
    RooWorkspace work;
    work.import(model);
    RooJSONFactoryWSTool tool(work);
    tool.exportJSON("argus.json");
 };
+
+#include "RooRealVar.h"
+#include "RooSimultaneous.h"
+#include "RooProdPdf.h"
+#include "RooCategory.h"
+
+TEST(RooFitHS3, SimultaneousGaussians)
+{
+   using namespace RooFit;
+
+   // Import keys and factory expressions files for the RooJSONFactoryWSTool.
+   std::string rootetcPath = gSystem->Getenv("ROOTSYS");
+   RooJSONFactoryWSTool::loadExportKeys(rootetcPath + "/etc/RooFitHS3_wsexportkeys.json");
+   RooJSONFactoryWSTool::loadFactoryExpressions(rootetcPath + "/etc/RooFitHS3_wsfactoryexpressions.json");
```

Comment:
```suggestion
   auto etcDir = std::string(TROOT::GetEtcDir());
   RooJSONFactoryWSTool::loadExportKeys(etcDir + "/RooFitHS3_wsexportkeys.json");
   RooJSONFactoryWSTool::loadFactoryExpressions(etcDir + "/RooFitHS3_wsfactoryexpressions.json");
```

The `etc` path is not so easy to construct as I thought. If you installed ROOT, it's `$ROOTSYS/etc/root`, but if you just have in uninstalled in a build directory it's just `$ROOTSYS/etc`. I have figured out that the right way to get the path is with `TROOT::GetEtcDir()`, so I suggest to do that (please make this change also in the test above). Note that this requires including `TROOT.h`.